packages/compass-indexes/src/modules/drop-index/index.js

```diff
@@ -80,6 +83,7 @@ export const dropIndex = (indexName) => {
     dispatch(toggleInProgress(true));
     state.dataService.dropIndex(ns, indexName, (err) => {
       if (!err) {
+        track('Index Dropped', {});
```

Comment:
it’s optional, but you can drop the second argument if you like (here and at the other sites which don’t take a properties argument):

```suggestion
        track('Index Dropped');
```